pkg/bundles/kontext/bundle.go

```diff
@@ -50,10 +52,23 @@ func bundle(directory string) (v1.Layer, error) {
 				return err
 			}
 
-			// Skip anything in the .git directory
-			// TODO(mattmoor): expand this to .gitignore / .dockerignore?
-			if fi.IsDir() && filepath.Base(path) == ".git" {
+			ignoreScanner, err := ignore.NewOrDefault(directory)
+
+			if err != nil {
+				return err
+			}
+
+			ignorable, err := ignoreScanner.CanIgnore(path, fi)
+
+			if err != nil {
+				return err
+			}
```

Comment:
I was puzzled by the `err` possibility and looking below I don't see any error paths.  I wouldn't expect this to need to surface error, did you have something in mind?

Reply:
this could be potentially ones from the ignorer